src/client/java/teammates/client/scripts/DataMigrationForEntityAttributes.java

```diff
@@ -0,0 +1,22 @@
+package teammates.client.scripts;
+
+import teammates.common.datatransfer.attributes.EntityAttributes;
+
+/**
+ * Base script for all data migration scripts involving {@link EntityAttributes}.
+ *
+ * @param <T> The entity attributes type to be migrated by the script.
+ */
+public abstract class DataMigrationForEntityAttributes<T extends EntityAttributes<?>> extends DataMigrationBaseScript<T> {
```

Comment:
I can imagine a use case for data migration of raw entities (without going through the attributes) though. I would name that script `DataMigrationForEntities`, and it would be incompatible with this one. Should I drop the word `For`? So just `DataMigrationEntityAttributes` and `DataMigrationEntities`?